src/libraries/Native/AnyOS/System.Globalization.Native/pal_icushim.c

```diff
@@ -243,13 +250,31 @@ static int FindICULibs(const char* versionPrefix, char* symbolName, char* symbol
 }
 
 #endif // __APPLE__
+#else // TARGET_UNIX
+
+// Windows implementation for FindICULibs
+static int FindICULibs()
+{
+    libicuuc = LoadLibraryExW(L"icu.dll", NULL, 0);
+    if (libicuuc == NULL)
+    {
+        return FALSE;
+    }
+
+    // Windows has a single dll for icu.
+    libicui18n = libicuuc;
+    return TRUE;
+}
+
+#endif // TARGET_UNIX
 
 // GlobalizationNative_LoadICU
 // This method get called from the managed side during the globalization initialization.
 // This method shouldn't get called at all if we are running in globalization invariant mode
 // return 0 if failed to load ICU and 1 otherwise
 int32_t GlobalizationNative_LoadICU()
 {
+#ifdef TARGET_UNIX
```

Comment:
It would look better to use #if / #elif / #else / #endif for this. Like:

`
#if defined(TARGET_WINDOWS)
... special Windows thing ...
#elif defined(TARGET_OSX)
... special Apple thing ...
#else
...
#endif
```
```

Reply:
Ok sounds good. Will update.